Makefile

```diff
@@ -63,8 +63,8 @@ npm-packages: lerna-build
 	$(call PACKAGE_LAB_EXTENSION,notebook-scheduler)
 	$(call PACKAGE_LAB_EXTENSION,pipeline-editor)
 	$(call PACKAGE_LAB_EXTENSION,python-runner)
-	cd dist && curl -O $$(npm view @jupyterlab/git@0.9.0 dist.tarball --userconfig=./npm_config) && cd -
-	cd dist && curl -O $$(npm view @jupyterlab/toc@2.0.0 dist.tarball --userconfig=./npm_config) && cd -
+	cd dist && curl -o jupyterlab-git-0.9.0.tgz $$(npm view @jupyterlab/git@0.9.0 dist.tarball) && cd -
+	cd dist && curl -o jupyterlab-toc-2.0.0.tgz $$(npm view @jupyterlab/toc@2.0.0 dist.tarball) && cd -
```

Comment:
Why are we changing these?

Reply:
after the binder build, after starting up lab, an error pops up requesting another rebuild stating the only reason being the git extension tar name is different than the one it currently had (`git-0.9.0.tgz` instead of `jupyterlab-git-0.9.0.tgz`). Slightly altering name was quickest fix and seemed trivial. Same version, just different filename.